doc/images.md

```diff
@@ -43,7 +43,19 @@ All render methods from the Sprite class can receive a `Paint` instance on the o
 
 Sprites can also be used as widgets, to do so, just use `Flame.util.spriteAsWidget`
 
-A complete example of using sprite as widgets can be found [here](examples/animation_widget).
+A complete example of using sprite as widgets can be found [here](/doc/examples/animation_widget).
+
+## SpriteBatch
+
+If you have an image atlas ( an image with smaller images inside ), and would like to render it effectively - `SpriteBatch` does the job.
```

Comment:
1. please don't add spaces inside parenthesis (like this).
2. we pretty much call "sprite atlas" spritesheets everywhere else. AFAICS this is the same thing just being rendered more efficiently. so maybe I'd change the text to be
> If you have a sprite sheet (also called an image atlas, which is ...), ...
just to make it more clear that spritesheet and atlas from the flame perspective are the same thing, this is just a new way to render it (correct me if I'm wrong)

Reply:
Yeah, they are essentially doing the same thing, only SpriteSheet now has a bit more functionality. Should replace "SpriteSheet" with "SpriteBatch" later ( and perhaps rename SpriteBatch to SpriteSheet again )